admin/services/check-window.service.js

```diff
@@ -1,14 +1,63 @@
 'use strict'
+const moment = require('moment')
 const CheckWindow = require('../models/check-window')
 
 const checkWindowService = {
-  getCurrentCheckWindow: async function () {
+
+  /**
+   * Get check window for the current time.
+   * @returns {Promise.<*>}
+   */
+  getCurrentCheckWindow: async () => {
     const now = new Date()
     const checkWindow = await CheckWindow.findOne({startDate: {$lte: now}, endDate: {$gte: now}}).exec()
     if (!checkWindow) {
       throw new Error('No checkwindow is currently available')
     }
     return checkWindow
+  },
+  /**
+   * Format check period (start and end dates).
+   * @param startDate
+   * @param endDate
+   * @returns {string}
+   */
+  formatCheckPeriod: (startDate, endDate) => {
```

Comment:
Date formatting should go in the DateService.

Reply:
@jon-shipley Did you notice how specific this formatting is? Do you still want it as a generic solution in the date service?

Reply:
@jon-shipley This custom date formatting is in the service.

Reply:
Okay